ios/BT/ExposureManager.swift

```diff
@@ -135,37 +137,40 @@ final class ExposureManager: NSObject {
         }
 
         dispatchGroup.notify(queue: .main) {
-          for result in localURLResults {
+          for result in localCompressedURLResults {
             switch result {
             case let .success(localURL):
-              localURLs.append(localURL)
+              localCompressedURLs.append(localURL)
             case let .failure(error):
               finish(.failure(error))
               return
             }
           }
-          
-          // TODO: Fetch configuration from API
-          let enConfiguration = ExposureConfiguration.placeholder.asENExposureConfiguration
-          ExposureManager.shared.manager.detectExposures(configuration: enConfiguration, diagnosisKeyURLs: localURLs) { summary, error in
-            if let error = error {
-              finish(.failure(error))
-              return
-            }
-            let userExplanation = NSLocalizedString("USER_NOTIFICATION_EXPLANATION", comment: "User notification")
-            ExposureManager.shared.manager.getExposureInfo(summary: summary!, userExplanation: userExplanation) { exposures, error in
+
+          localCompressedURLs.decompress { (uncompressedUrls) in
+            localUncompressedURLs = uncompressedUrls
+            // TODO: Fetch configuration from API
+            let enConfiguration = ExposureConfiguration.placeholder.asENExposureConfiguration
+            ExposureManager.shared.manager.detectExposures(configuration: enConfiguration, diagnosisKeyURLs: uncompressedUrls) { summary, error in
               if let error = error {
                 finish(.failure(error))
                 return
               }
-              let newExposures = exposures!.map { exposure in
-                Exposure(id: UUID().uuidString,
-                         date: exposure.date.posixRepresentation,
-                         duration: exposure.duration,
-                         totalRiskScore: exposure.totalRiskScore,
-                         transmissionRiskLevel: exposure.transmissionRiskLevel)
+              let userExplanation = NSLocalizedString("USER_NOTIFICATION_EXPLANATION", comment: "User notification")
+              ExposureManager.shared.manager.getExposureInfo(summary: summary!, userExplanation: userExplanation) { exposures, error in
```

Comment:
we ended up introducing a concept of `ExposureInfo` on the client side as well, to name the map of date to aggregated exposures for a user.  What does ExposureInfo mean here? If it means something different here, should we update the name on the client side?